core/pva/src/main/java/org/epics/pva/client/MonitorRequest.java

```diff
@@ -159,6 +159,11 @@ public void handleResponse(final ByteBuffer buffer) throws Exception
 
                 // Decode type description from INIT response
                 final PVAData type = channel.getTCP().getTypeRegistry().decodeType("", buffer);
+                byte[] b = new byte[buffer.limit()];
```

Comment:
This printout in the PVA MonitorRequest looks like debug code that shouldn't be merged

Reply:
You are quite right, I will fix.